File: system/src/main/scala/com/anjunar/jaxrs/types/Sort.scala
package com.anjunar.jaxrs.types

import com.anjunar.scala.mapper.annotations.PropertyDescriptor
import scala.beans.BeanProperty
import scala.compiletime.uninitialized

class Sort {

  @PropertyDescriptor(title = "Property")
  var property : String = uninitialized

  @PropertyDescriptor(title = "Value")
  var value : String = uninitialized

}

File: application/src/main/javascript/workspace/projects/shared/src/lib/domain/container/Sort.ts
import {Entity} from "../../mapper";
import ActiveObject from "./ActiveObject";
import Primitive from "../../mapper/annotations/Primitive";

@Entity("Sort")
export default class Sort extends ActiveObject {

    override $type = "Sort"

    @Primitive()
    property : string

    @Primitive()
    value : string

}